src/components/TradeWidget/TxMessage.tsx

```diff
@@ -30,25 +31,49 @@ export const TxMessage: React.FC<TxMessageProps> = ({ sellToken, receiveToken })
 
   return (
     <TxMessageWrapper>
-      <p>
-        Your are selling {sellTokenAmount} {displaySellToken} for {displayReceiveToken}
-      </p>
-      <p>At a price of</p>
-      <p>
-        {priceInverse} {displaySellToken} per {displayReceiveToken}
-      </p>
-      <p>
-        {price} {displayReceiveToken} per {displaySellToken}
-      </p>
-      <p>
-        <strong>
-          You will receive at least {receiveTokenAmount} {displayReceiveToken}
-        </strong>
-      </p>
-      <p>
-        Your order starts {formatTimeInHours(validFrom || 0, 'now')}{' '}
-        {validUntil && `and will expire ${formatTimeInHours(validUntil, 'Never')}`}
-      </p>
+      <div className="intro-text">
+        <div>Carefully review the information below to make sure everything looks correct.</div>
+      </div>
+      <div className="message">
+        {/* Details */}
+        <div>
+          <strong>Order Details</strong>
+        </div>
+        <div>
+          Sell: <span>{sellTokenAmount}</span> <strong>{displaySellToken}</strong> for{' '}
+          <strong>{displayReceiveToken}</strong>
+        </div>
+        <div>
+          Receive: <UnderlinedText>at least</UnderlinedText> {receiveTokenAmount} <strong>{displayReceiveToken}</strong>
+        </div>
+
+        {/* Prices */}
+        <p>
+          <div>
+            <strong>Prices</strong>
+          </div>
+          <div>
+            Direct: <span>{priceInverse}</span> <strong>{displaySellToken}</strong> per{' '}
+            <strong>{displayReceiveToken}</strong>
+          </div>
+          <div>
+            Inverse: <span>{price}</span> <strong>{displayReceiveToken}</strong> per <strong>{displaySellToken}</strong>
```

Comment:
I think it's strange. Let's just use the new approach we are using in other parts of the app. 
**Offer one price, and allow to invert it**. 

The format is nice (the TOKEN1 pero TOKEN2), but you are not coherent with the web, where we are currently showing BUY/SELL as default. So sell token is the quote. For the screenshot, it should say WETH per GUSD

> IMPORTANT: Please, let's keep out of the discussion the other change we have pending for analysing that uses the stable coin as the quote. I agree it should be the opposite to what I said above once we address this, but for now, it should be always BUY/SELL market

David. If you make a component for a price, that shows a price, the units an allows to invert it we can start using it everywhere and the moment we improve the quote token selection it will be easier to update

@Rafanator